conf/modules.config

```diff
@@ -1,41 +1,247 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Config file for defining DSL2 per module options and publishing paths
-========================================================================================
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Available keys to override module options:
-        ext.args            = Additional arguments appended to command in module.
-        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
-        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
-        ext.prefix          = File name prefix for output files.
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
 ----------------------------------------------------------------------------------------
 */
 
 process {
 
     publishDir = [
         path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
-        mode: 'copy',
+        mode: params.publish_dir_mode,
         saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
     ]
 
-    withName: SAMPLESHEET_CHECK {
+    withName: FASTQC {
+        ext.args = '--quiet'
         publishDir = [
-            path: { "${params.outdir}/pipeline_info" },
+            [
+                path: { "${params.outdir}/fastqc" },
+                mode: 'copy',
+                pattern: "*.html"
+            ]
+        ]
+    }
+
+    withName: TRIMGALORE {
+        ext.args = [
+            '--fastqc',
+            params.trim_nextseq > 0 ? "--nextseq ${params.trim_nextseq}" : ''
+        ].join(' ').trim()
+        publishDir = [
+            [
+                path: { "${params.outdir}/trimgalore/fastqc" },
+                mode: 'copy',
+                pattern: "*.html"
+            ],
+            [
+                path: { "${params.outdir}/trimgalore" },
+                mode: 'copy',
+                pattern: "*.txt"
+            ]
+        ]
+    }
+
+    withName: GUNZIP {
+        publishDir = [
+            enabled: false
+        ]
+    }
+    withName: GUNZIP_GFFS {
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: 'FIRST_CAT' {
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: SECOND_CAT {
+        publishDir = [
+            path: { "${params.outdir}/concatenate" },
             mode: 'copy',
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            pattern: '*.gz',
+            enabled: params.save_genomes_cat
         ]
     }
 
```

Comment:
Where is the third cat?!

Reply:
but more seriously, I don't find these module names informative, I had to search where you used it... hy not `CAT_GFF_ONE`, `CAT_GFF_TWO` or something?

Reply:
Not informative as not everyone may know what `cat` is

Reply:
Or might get confused with https://github.com/MGXlab/CAT_pack

Reply:
I suppose `CONCATENATE` would be more transparent, followed by file type (and number). (I think the number is overplayed by recent changes, actually.)